package.json

```diff
@@ -38,14 +38,8 @@
   "author": "Plaid",
   "license": "MIT",
   "peerDependencies": {
-    "react": "~16.13.1",
-    "react-native": "^0.63.3"
-  },
-  "dependencies": {
-    "@typescript-eslint/parser": "^4.6.1",
-    "prop-types": "^15.7.2",
```

Comment:
Was this dependency unused?

Reply:
Yeah we were actually never importing this package:
https://www.npmjs.com/package/prop-types